apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/index-table.tsx

```diff
@@ -52,6 +52,8 @@ export const IndexTable = <T extends IBaseViewModel>({
     heading,
     filterableProperties,
 }: GenericIndexTablePresenterProps<T>) => {
+    const theme = useTheme();
+
     if (headingLabels.length === 0) {
```

Comment:
Does this belong in a presenter? It seems like a state issue. This is my previous code that needs to be revisited btw.

Could the presenter be the one to display a "no data" type message?